soda.rb

```diff
@@ -1,5 +1,5 @@
 class Soda
-  attr_reader :brand, :price
+  attr_reader :brand, :price, :sodas
```

Comment:
Why does an individual soda have a `sodas` reader? It doesn't have a `sodas` instance variable, so the getter will always return nil!